Node method unit tests still failing

Two of the unit tests regarding the node method and node method mass flow calculation are failing: 
```
======================================================== short test summary info ========================================================
FAILED modesto/Tests/test_examples.py::test_example_node_method - IndexError: too many indices for array: array is 1-dimensional, but 2 were indexed
FAILED modesto/Tests/test_mf_calculation.py::test_mass_flow_calculation - IndexError: too many indices for array: array is 1-dimensional, but 2 were indexed

============================================== 2 failed, 32 passed, 10 warnings in 22.95s ===============================================
```

Unsure why this is the case, and I'm unfamiliar with this part of the code. To be solved if somebody needs the node method for calculations.

--- a/modesto/Tests/test_mf_calculation.py
+++ b/modesto/Tests/test_mf_calculation.py
@@ -68,3 +68,7 @@ def test_mass_flow_calculation():
 
     if not flag:
         raise Exception('The mass flow calculation was incorrect')
+
+
+if __name__ == '__main__':
+    test_mass_flow_calculation()

--- a/modesto/mass_flow_calculation.py
+++ b/modesto/mass_flow_calculation.py
@@ -135,7 +135,7 @@ class MfCalculation(object):
 
             # Calculate mass flow through producer node
             unknown_node_mf = (sum(
-                self.mass_flows[edge][t] * row[0, i] for i, edge in
+                self.mass_flows[edge].iloc[t] * row[i] for i, edge in
                 enumerate(self.edges)))
 
             # Calculate mass flow through producer component

--- a/modesto/mass_flow_calculation.py
+++ b/modesto/mass_flow_calculation.py
@@ -97,9 +97,9 @@ class MfCalculation(object):
 
     def initialize_mass_flows(self):
         for edge in self.edges:
-            self.mass_flows[edge] = pd.Series(0, index=self.index)
+            self.mass_flows[edge] = pd.Series(0, index=self.index, dtype='float64')
 
-        self.mass_flows[self.unknown_node][self.unknown_comp] = pd.Series(0, index=self.index)
+        self.mass_flows[self.unknown_node][self.unknown_comp] = pd.Series(0, index=self.index, dtype='float64')
 
     def calculate_mf(self):
         """

